custom_components/tuya_local/devices/poolex_arctic_action.yaml

```diff
@@ -0,0 +1,267 @@
+          icon_priority: 2
+        - dps_val: 3
+          value: "Water Flow Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 4
+          value: "Frost Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 5
+          value: "High Pressure Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 6
+          value: "Low Pressure Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 9
+          value: "Main Board - Line Control Communication Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 10
+          value: "Frequency Module and Main Board Communication Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 12
+          value: "Exhaust Over High Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 13
+          value: "DC Fan Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 15
+          value: "Inlet Water Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 16
+          value: "External Coil Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 18
+          value: "Exhaust Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 20
+          value: "Frequency Module Abnormal Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 21
+          value: "Ambient Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 23
+          value: "Cooling Water Temperature Too Low Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 27
+          value: "Outlet Water Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 29
+          value: "Return Gas Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 32
+          value: "Heating Water Temperature Too High Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 33
+          value: "Outdoor Coil High Temperature Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 35
+          value: "Compressor Current Protection"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+        - dps_val: 42
+          value: "Internal Coil Temperature Fault"
+          icon: "mdi:water-boiler-alert"
+          icon_priority: 2
+
+secondary_entities:
+  - entity: climate
+    icon: "mdi:hot-tub"
+    deprecated: climate
```

Comment:
In my previous comment, the intention was to suggest that you remove this deprecated entity, not change it to a second climate entity.